src/Button/index.scss

```diff
@@ -0,0 +1,36 @@
+@use "sass:map";
+
+@use "../sass/mixins";
+@use "../sass/tokens";
+
+.button {
+  display: inline-flex;
+  flex-direction: row;
+
+  margin: 0;
+  border: 0;
+
+  font-family: map.get(tokens.$fonts, sans);
+
+  @include mixins.transition(color, medium);
```

Comment:
Make sure to also transition `background-color`. When I hover over the blue buttons especially the lack of transition makes it look a bit jarring.